### ISSUE TITLE
Question regarding normailzation

### ISSUE THREAD

**Issue by austenmyers** [austenmyers/TensorFlow-Pokemon-Course#3]
Question regarding normailzation:
Shouldn't we fit_transform() the scaler once on the training set, and then call transform() on the test set?

In this example, the method call fit_transform twice, on the train set and the test set. Is this correct?
